src/Components/Carousel.js

```diff
@@ -3,14 +3,20 @@ import GameCard from './GameCard';
 import { uid } from 'react-uid';
 
 class Carousel extends Component {
+
+  shiftCarousel(event) {
+    const shiftDirection = event.target.classList.contains('fa-angle-right') ? 1 : -1;
+    const currentPosition = event.target.parentElement.childNodes[2].scrollLeft;
+    event.target.parentElement.childNodes[2].scrollLeft = currentPosition + (250 * shiftDirection);
+  }
```

Comment:
Reassigning the position of the element 250px left or right depending on which button was clicked

Reply:
This is great, nice work Matt. I hadn't used the scrollLeft property yet so it's cool to see it in action.